test/unit/requirements.txt

```diff
@@ -1,4 +1,5 @@
-pytest
```

Comment:
Is there an upper limit on supported version for any packet? I remember that we had some issue with pytest > 6.0.

Reply:
No, I am not aware of any upper limit. There was indeed an issue with recent pytest, but that has been fixed (worked around) since.